### PR TITLE
Devops 8: Create a front-end project

### PR DESCRIPTION
**Pull request by frenchal17**
Created an Angular 5 front end project with a simple back-end endpoint call.
Updated README to reflect UI compilation and serving instructions.

### CODE REVIEW

**Comment by frenchal17** [comment]
CorsConfig related files were necessary to allow Cross-origin requests

File: devops-accounts-service/src/main/java/app/devops/controllers/AccountController.java [hunk 1]
```diff
@@ -37,8 +40,11 @@ public String addAccount(@RequestBody AccountDTO accountDto) {
 		return "Account with username \"" + accountDto.getUserName() + "\" added successfully!";
 	}
 
+	@CrossOrigin(origins = "http://localhost:4200")
 	@RequestMapping("/getHello")
-	public String getHello(){
-		return "Frenchie is crying, because Nate is spending too much money!";
+	public String getHello() throws JSONException{
+		JSONObject object = new JSONObject();
+		object.put("data", "Frenchie is crying, because Nate is spending too much money!");
```
Comment (RussianPrussian) [inline devops-accounts-service/src/main/java/app/devops/controllers/AccountController.java]: This is going to break the integration tests. Can you pull in DEVOPS-31 and run the AccountsIntTests. You'll have to fix the test on this method.

Comment (frenchal17) [reply devops-accounts-service/src/main/java/app/devops/controllers/AccountController.java]: I'd noticed that -- I had been planning on fixing the test before PR but had forgotten about it. Will do that soon.

**Comment by lenardw1** [comment]
@frenchal17  you can pull the integration test fixes from DEV now

**Pull request merged**